app/src/main/java/com/example/ithaca_transit_android_v2/util/CurrLocationManager.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.ithaca_transit_android_v2.util
+
+import android.Manifest
+import android.app.Activity
+import android.content.Context
+import android.content.pm.PackageManager
+import android.location.LocationManager
+import androidx.core.app.ActivityCompat
+
+class CurrLocationManager(c: Context, locationManager: LocationManager, activity: Activity) {
+    var mContext: Context? = null
```

Comment:
since `c` is not nullable, `mContext` will never be nullable

Reply:
good point